opentracing/tracer.py

```diff
@@ -40,31 +41,84 @@ class Tracer(object):
     def __init__(self):
         self._noop_span_context = SpanContext()
         self._noop_span = Span(tracer=self, context=self._noop_span_context)
+        self._active_span_source = NoopActiveSpanSource()
 
-    def start_span(self,
-                   operation_name=None,
-                   child_of=None,
-                   references=None,
-                   tags=None,
-                   start_time=None):
+    @property
+    def active_span_source(self):
+        return self._active_span_source
+
+    @property
+    def active_span(self):
+        return self._active_span_source.active_span
+
+    def start_active_span(self, operation_name, tags=None, start_time=None):
```

Comment:
Sure, I misunderstand that part of the API. I'm going to change that as you've suggested! Thanks.